licences-specs/src/test/groovy/uk/gov/justice/digital/hmpps/licences/specs/pdf/PdfSpec.groovy

```diff
@@ -37,6 +37,7 @@ class PdfSpec extends GebReportingSpec {
 
     then: 'There is no option to create PDF'
     !taskListAction('Create licence').isDisplayed()
+    actions.logOut()
```

Comment:
needed as the test tries to login as two people without logging out.  This doesn't work when auth remembers who you are after login